sync.go

```diff
@@ -491,3 +493,16 @@ func (mw *MultiWallet) GetLowestBlockTimestamp() int64 {
 	}
 	return timestamp
 }
+
+func (mw *MultiWallet) GetActivationBlockHeight() int32 {
```

Comment:
- This should go to the multiwallet utils file.
- [Go methods should not be prefixed by `Get`](https://go.dev/doc/effective_go#Getters), just `ActivationBlockHeight` is recommended.
- Also, `ActivationBlockHeight` isn't descriptive, use `DCP0001ActivationBlockHeight`. Also add a comment that says `returns the hardcoded block height that the DCP0001 deployment activates at. DCP0001 specifies hard forking changes to the stake difficulty algorithm.`.